src/displayCategory.js

```diff
@@ -0,0 +1,8 @@
+import { displayOne } from './displayOne';
+
+// Display of all categories on page
+export function updateUI_categoryPage(arr) {
```

Comment:
This function could be in same file as displayOne. Try to put all this UI related functions into same file, so to be less files, and more clear project structure. Avoid broken camel case naming.